### DISCUSSION TITLE
Should we enable merge queues?

### DISCUSSION THREAD

**Discussion by jamie-o-wilkinson** [govuk-forms/forms#94] [Polls]
> [About merge queues](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue#about-merge-queues)
> A merge queue helps increase velocity by automating pull request merges into a busy branch and ensuring the branch is never broken by incompatible changes.
> 
> The merge queue provides the same benefits as the Require branches to be up to date before merging branch protection, but does not require a pull request author to update their pull request branch and wait for status checks to finish before trying to merge.
> 
> Using a merge queue is particularly useful on branches that have a relatively high number of pull requests merging each day from many different users.
> 
> Once a pull request has passed all required branch protection checks, a user with write access to the repository can add the pull request to the queue. The merge queue will ensure the pull request's changes pass all required status checks when applied to the latest version of the target branch and any pull requests already in the queue.

**Comment by DavidBiddle** [comment]
Would work nicely with #44
